lib/s3-html/log.html

```diff
@@ -190,7 +190,7 @@
     {
         "label": "coverage",
         "pattern": "Code coverage report in ([A-Za-z0-9\\-\\.]+)$",
```

Comment:
I guess this isn't going to work: that regexp doesn't match `/`.

Reply:
Yes, thanks for catching that!